ArgoProject/src/systems/CollisionSystem.cpp

```diff
@@ -223,6 +226,26 @@ void app::sys::CollisionSystem::checkAINodeCollisions()
 	});
 }
 
+void app::sys::CollisionSystem::attackEnemyCollisions()
+{
+	//look through all attacks
+	m_registry.view<comp::Collision, comp::Attack, comp::Location, comp::Dimensions, comp::Damage>(entt::persistent_t())
```

Comment:
make sure there is the appropriate `m_registry.prepare` call in the systems constructor (note the other of components must be the same)